More brig/wanderer jobs (#6044)
endless-sky/endless-sky: data/wanderer/wanderer jobs.txt

	on complete
		payment 35000 300
		dialog `When you return, the Wanderers thank you for taking them to the Kor Efreti and hand you <payment> before disembarking.`

mission "Captured soldiers"
	name "Transport captured Unfettered soldiers"
	job 
	repeat

> 	description "Transport <bunks> captured Unfettered soldiers to <planet>, payment is <payment>"

Reviewer: ```suggestion
	description "Transport <bunks> captured Unfettered soldiers to <planet>, payment is <payment>"
	to offer
		has "language: Wanderer"
		random < 20
```

	passengers 3
	on offer
		require "Brig"

> 	source
> 		has "language: Wanderer"
> 		random < 20
> 		government "Wanderer"

Reviewer: ```suggestion
	source
		government "Wanderer"
```

	destination
		distance 2 8
		government "Wanderer"
	on complete
		payment 50000

> 		dialog "You wish the Wanderers the best of luck on <planet> and collect your payment of <payment>."

Reviewer: You're not carrying Wanderers, so this line makes no sense.
```suggestion
		dialog "Once you land on <planet>, you watch as some Wanderers in uniform carry the Unfettered prisoners off of your ship. You collect your payment of <payment>."
```
